### DISCUSSION TITLE
Any way to transmit usage logs to Datadog?

### DISCUSSION THREAD

**Discussion by pradip-eliza** [deepgram/community#633] [General help] [answered]
Want to get some alerting going on when our requests/callbacks fail and improving vis for this. Is there a way to move usage logs to Datadog or the like?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@pradip-eliza Sure you could definitely get usage data from our API (see our [docs)](https://developers.deepgram.com/reference/get-all-requests) and programmatically pass that to DataDog. I suggest you checkout DataDog's Docs for more info but getting Usage Data out of Deepgram is possible via our API.
